addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/handler/GlobalCacheHandler.java

```diff
@@ -264,7 +272,7 @@ private String lookupCode(Command command, ChannelUID channelUID) {
                 code = convertHexToGC(code);
                 logger.debug("Converted hex code is: {}", code);
             } catch (HexCodeConversionException e) {
-                logger.warn("Failed to convert hex code to GC format");
+                logger.info("Failed to convert hex code to GC format");
```

Comment:
I think it should stay as info, as it's a user "data" issue.  The user can put two different IR code formats in the MAP file - either a hex IR code or a globalcache IR code.  If it's a hex IR code, it needs to be converted to globalcache format using the method `convertHexToGC`.  That conversion could fail for a number of reasons, all of which will generate a `HexCodeConversionException`.  I do need to include the message from the exception, so that the user knows what aspect of the hex code caused the conversion failure so that they can fix it.  Ok?